Essentials/src/main/resources/messages_es.properties

```diff
@@ -1419,6 +1419,8 @@ whoisGod=<gray> - Modo dios\:<white> {0}
 whoisHealth=<gray> - Salud\:<white> {0}/20
 whoisHunger=<gray> - Hambre\:<reset> {0}/20 (+{1} saturación)
 whoisIPAddress=<gray> - Dirección IP\:<white> {0}
+whoisLastLogin=<gray> - Último inicio de sesión\:<white> {0}
+whoisFirstLogin=<gold> - Primer inicio de sesión\:<white> {0}
```

Comment:
Please don't add translations to any file except `messages.properties` inside PRs. Translations to other languages are handled via Crowdin after a PR is submitted. Any strings you add to language-specific translation files (including `en` and `es`) will be deleted by Crowdin.

(You also appear to have used different formatting here to the source string in `messages.properties`.)